src/index.ts

```diff
@@ -0,0 +1,10 @@
+/**
+ * index.ts - Loads the WoT Adapter.
+ *
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/.*
+ */
+
+export * from './wot-adapter'
```

Comment:
I don't think this is going to work.
The gateway expects a single export which is the load function.

Reply:
Right, solve in b750193